themes/tiddlywiki/vanilla/base.tid

```diff
@@ -47,6 +47,20 @@ $else$
 </$reveal>
 \end
 
+\define set-type-selector-min-width()
+<$list filter="[all[shadows+tiddlers]removeprefix[$:/language/Docs/Types/]length[]] +[nsort[]] +[last[]]" variable="typeLength">
```

Comment:
`[all[shadows+tiddlers]removeprefix[$:/language/Docs/Types/]length[]maxall[]]` does the same thing BUT, that's not the right size. the text is combined `name` + `description`  field if I remember right.

There probably is a much bigger performance problem, but I'm not sure without testing

Reply:
@pmario , you're right, it's the `name` field that gets used

The performance impact cannot be that high (I hope) :smile:

Reply:
Using the `maxall[]` operator on the length gives me `-Infinity` which is strange

Reply:
Ah sorry! I found my error